VKApp/VKApp/CustomUIComponents/CustomNavigationController.swift

```diff
@@ -0,0 +1,58 @@
+// CustomNavigationController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Наш кастомный UINavigationController который наполнен анимациями
```

Comment:
```suggestion
/// Наш кастомный UINavigationController который наполнен анимациями
```
Наш отечественный)? наполнен солнцем и изобилием)? Дай нормальное название доке